<?xml version="1.0"?>
<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find manipulator)/config/ros_controllers.yaml" command="load"/>

  <!-- start this controller -->
  <node name="arm_controller_spawner" pkg="controller_manager" type="controller_manager" args="spawn blue_arm_controller joint_state_controller" respawn="false" output="screen"/>

  <!-- load other controllers -->
  <!-- <node name="ros_control_controller_manager" pkg="controller_manager" type="controller_manager" respawn="false" output="screen" args="load joint_group_position_controller" /> -->

  <!-- <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
    <param name="publish_frequency" type="double" value="50.0" />
    <param name="tf_prefix" type="string" value="" />
  </node> -->

  <!-- joint_state_controller -->
  <!-- <rosparam file="$(find manipulator)/config/ros_controllers.yaml" command="load"/> -->
  <!-- <node name="joint_state_controller_spawner" pkg="controller_manager" type="controller_manager" args="spawn joint_state_controller" respawn="false" output="screen"/> -->


</launch>
